### PR TITLE
hw 6 with loops  was added

### PR DESCRIPTION
**Pull request by serhii-k-n**

### CODE REVIEW

File: Homework_6/loops.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+users_number = int(input('Print your number here: '))
+
+for number_first in first_list_of_numbers:
+    if number_first % users_number == 0:
+        divide_list_of_numbers.append(number_first)
+
+if divide_list_of_numbers:
+    print(f'{users_number} is divisible by a list: {divide_list_of_numbers} without a remainder')
+else:
+    print(f'{users_number} is not divisible by any number from the list: {first_list_of_numbers} without a remainder.')
```
Comment (MarinaZelenska) [inline Homework_6/loops.py]: I suppose if block here is extra

File: Homework_6/loops.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+for number_second in second_list_of_numbers[1:]:
+    if number_second != expected_number:  # If the current number is not equal to the expected number
+        print(f'\nFirst non-consecutive number: {number_second}\n')
+        found_gap = True
+        break
+    prev_number_from_list = number_second
+    expected_number += 1
+
+if not found_gap:
+    print(f'\nThe list: {second_list_of_numbers} consists of consecutive numbers\n')
```
Comment (MarinaZelenska) [inline Homework_6/loops.py]: Work, but difficult
image

**Review by MarinaZelenska — APPROVED** [review]

File: Homework_6/loops.py (hunk 1 above)
Comment (serhii-k-n) [reply Homework_6/loops.py]: done

File: Homework_6/loops.py (hunk 2 above)
Comment (serhii-k-n) [reply Homework_6/loops.py]: done

**Pull request merged**